### PR TITLE
Con-942: Clean up navigation Light/Dark theme in Mobile

### PR DESCRIPTION
**Pull request by cyrusxzw**

### CODE REVIEW

File: src/components/GlobalNavigation/components/MobileView/components/Header/Header.module.css [hunk 1]
```diff
@@ -83,23 +80,69 @@
...
-    color: var(--color-secondary);
+    color: var(--color-primary);
   }
 
   .light &,
   .lightClosed & {
+    color: var(--color-secondary);
+  }
+
+  /* Added new */
```
Comment (hassanraza0054) [inline src/components/GlobalNavigation/components/MobileView/components/Header/Header.module.css:91]: You can probably remove this comment now!

File: src/components/GlobalNavigation/components/MobileView/components/Header/Header.module.css [hunk 2]
```diff
@@ -83,23 +80,69 @@
...
   .darkClosed & {
-    background-color: var(--color-secondary);
+    background-color: var(--color-primary);
   }
 
   .lightClosed & {
+    background-color: var(--color-secondary);
+  }
+
+  /* Added new */
```
Comment (hassanraza0054) [inline src/components/GlobalNavigation/components/MobileView/components/Header/Header.module.css:138]: Same here!

File: src/components/GlobalNavigation/components/MobileView/components/Header/Header.module.css [hunk 3]
```diff
@@ -148,15 +191,26 @@
...
   }
 
   .light &::before,
   .light &::after,
   .lightClosed &::before,
   .lightClosed &::after {
+    background-color: var(--color-secondary);
+  }
+
+  /* Added new */
```
Comment (hassanraza0054) [inline src/components/GlobalNavigation/components/MobileView/components/Header/Header.module.css:204]: And another one haha

File: src/components/GlobalNavigation/components/MobileView/components/Header/Header.module.css [hunk 4]
```diff
@@ -165,12 +219,51 @@
...
+  .light & {
     border-top: 1px solid var(--color-grey-60);
     background-color: var(--color-primary);
   }
+}
 
-  .light & {
-    border-top: 1px solid var(--color-white);
-    background-color: var(--color-secondary);
+/* Overwrite sticky header in dark/light and light/dark theme */
```
Comment (hassanraza0054) [inline src/components/GlobalNavigation/components/MobileView/components/Header/Header.module.css:232]: Nice putting a comment here to contextualise 👍

File: src/components/GlobalNavigation/components/TabletView/TabletView.module.css [hunk 5]
```diff
@@ -43,6 +43,7 @@
   &.isVisibleStickyNav,
   &.isInvisibleStickyNav {
     position: fixed;
+    top: calc(var(--menu-height) * -1);
```
Comment (hassanraza0054) [inline src/components/GlobalNavigation/components/TabletView/TabletView.module.css:46]: Ah this is that separate change you made a bug ticket for. Lets make this change in another PR to keep the changes specific!

**Review by hassanraza0054 — APPROVED** [review]
All good! Just dropped a few indicators to remove comments otherwise we can go ahead with this.

File: src/components/GlobalNavigation/components/MobileView/components/Header/Header.module.css (hunk 1 above)
Comment (cyrusxzw) [reply src/components/GlobalNavigation/components/MobileView/components/Header/Header.module.css:91]: I may change it to a more clear comment.
